AUTHORS

```diff
@@ -185,7 +186,7 @@ Thomas Grainger <tagrain@gmail.com>
 Thomas Walpole <twalpole@gmail.com>
 timm-permeance <tim.myerscough@permeance.com.au>
 Timofey Vasenin <timofey.vasenin@gmail.com>
-Titus Fortner <titus.fortner@blackbaud.com>
+Titus Fortner <titus.fortner@gmail.com>
```

Comment:
this file is generated, and based on your .gitconfig.  this might be overridden in the future fyi :)